docker/bin/certify.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eu
+
+DOCKER_BIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+export PASS="abcdefgh"
```

Comment:
you don't need to export PASS from what I can see

Reply:
gen-ssl-certs.sh uses it, without the export you will get an unbound variable error